input/pagecontent/gaps-examples.md

```diff
@@ -19,14 +19,14 @@ Later, the provider runs the same Gaps in Care Report again, both patients now s
 |Bundle| DEQM Gaps In Care Bundle Profile        | [DEQM Gaps In Care Bundle Profile]        
 |Composition| DEQM Gaps In Care Composition Profile   | [DEQM Gaps In Care Composition Profile]   
 |DetectedIssue| DEQM Gaps In Care DetectedIssue Profile | [DEQM Gaps In Care DetectedIssue Profile] 
-|Encounter| QI Core Encounter Profile               | [QI Core Encounter]                       |
-|Library| CRMI Shareable Library                    | [CRMI Shareable Library]                            |
-|Measure| CRMI Shareable Measure Profile                    | [CRMI Shareable Measure]                            |
+|Encounter| QI-Core Encounter Profile               | [QI-Core Encounter]                       |
```

Comment:
These are specific QI-Core profiles. Change to "QICore" instead of "QI-Core" to match the actual profile names, e.g., QICore Encounter.